### PR TITLE
[OCPBUGS-2544]: IPsec offering is east/west for intra-cluster traffic

### PR DESCRIPTION
**Pull request by tmalove**
[OCPBUGS-2544](https://issues.redhat.com/browse/OCPBUGS-2544)

Version(s): 4.11+

Link to docs [preview](http://file.rdu.redhat.com/tlove/sdn-ocpbugs-2544-tlove/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.html#nw-ovn-kubernetes-matrix_about-ovn-kubernetes)  (Updated 10/25)

[x] QE has approved this change.

### CODE REVIEW

**Review by tmalove — COMMENTED** [review]
/label peer-review-needed

**Comment by CFields651** [comment]
LGTM!

**Comment by tmalove** [comment]
@zhaozhanqi  WIll you review this update? Thanks!

**Comment by zhaozhanqi** [comment]
> @zhaozhanqi Will you review this update? Thanks!

@anuragthehatter ^^

**Comment by anuragthehatter** [comment]
Thanks

/lgtm

File: modules/nw-ovn-kubernetes-matrix.adoc [hunk 1]
```diff
@@ -20,7 +20,7 @@ ifeval::["{context}" == "about-ovn-kubernetes"]
 
 |Egress router|Supported ^[2]^|Supported
 
-|IPsec encryption|Supported|Not supported
+|IPsec encryption for east/west traffic|Supported|Not supported
```
[the remark this answers is not in the document]
Comment (tmalove) [reply modules/nw-ovn-kubernetes-matrix.adoc:23]: Thanks @maxwelldb  ...I will look into this.

**Comment by tmalove** [comment]
@CFields651 @anuragthehatter  I need to eliminate the slashes per our style guide. I can change the table entry to 'IPsec encryption for internal data center traffic' or something else that is similar. Thanks!

**Comment by anuragthehatter** [comment]
Looks good @tmalove Thanks

**Comment by CFields651** [comment]
something to the effect of "intra-cluster communication" would replace east/west

**Comment by CFields651** [comment]
LGTM!

**Pull request merged**